pyk/src/pyk/kcfg.py

```diff
@@ -110,40 +113,40 @@ def __exit__(self, exc_type, exc_value, traceback):
         return False
 
     @property
-    def nodes(self) -> Set[Node]:
-        return set(self._nodes.values())
+    def nodes(self) -> List[Node]:
+        return list(self._nodes.values())
```

Comment:
Why do we return as `list` now instead of `set`? Is it so that we can do things like `cfg.nodes()[0]`?

Reply:
Primarily, but also because lists are cheaper to construct than sets.